maya/mel/assign_glTF_PBR_material_and_textures.mel

```diff
@@ -0,0 +1,80 @@
+global proc assign_glTF_PBR_material_and_textures(
```

Comment:
I would pass the regular expressions of the texture file-names as an argument, and make a `global $default_glTF_texture_patterns` or something. We can't hard-code these texture names, this is studio and tool dependent.